stdlib/asyncio/windows_utils.pyi

```diff
@@ -36,3 +36,23 @@ if sys.platform == "win32":
         def handle(self) -> int: ...
         def fileno(self) -> int: ...
         def close(self, *, CloseHandle: Callable[[int], None] = ...) -> None: ...
+
+    class Popen(subprocess.Popen[AnyStr]):
+        # The docstring says "The stdin, stdout, stderr are None or instances of PipeHandle."
+        # but that is incorrect: if you pass in a file object, that will still be
+        # put in the stdin/stdout/stderr attributes.
+        stdin: PipeHandle | IO[AnyStr] | None
+        stdout: PipeHandle | IO[AnyStr] | None
+        stderr: PipeHandle | IO[AnyStr] | None
```

Comment:
```suggestion
        stdin: PipeHandle | IO[AnyStr] | None  # type: ignore[override]
        stdout: PipeHandle | IO[AnyStr] | None  # type: ignore[override]
        stderr: PipeHandle | IO[AnyStr] | None  # type: ignore[override]
```

Reply:
Needs to be `[assignment]` rather than `[override]`

Reply:
Huh, thanks. That's a weird code to use there; I'm not even assigning anything.